docs/user-guide/get-started.md

```diff
@@ -74,65 +80,59 @@ If you use Prettier alongside Stylelint, you should use their [SCSS shared confi
 }
 ```
 
+Other shared configs include:
+
+- [stylelint-config-html](https://www.npmjs.com/package/stylelint-config-html)
+- [stylelint-config-recommended-vue](https://www.npmjs.com/package/stylelint-config-recommended-vue)
+
+### Using a custom syntax directly
+
 If a shared config isn't available for your preferred language or library, then you can install the appropriate [PostCSS syntax](https://github.com/postcss/postcss#syntaxes) yourself and use the [`customSyntax` option](../user-guide/usage/options.md#customSyntax) to configure it.
 
-For example, to lint [SugarSS](https://github.com/postcss/sugarss).
+For example, to lint the CSS inside of [Lit elements](https://lit.dev/).
```

Comment:
Switching out to postcss-lit from sugarss as:

- sugarss isn't popular
- lit elements are web components, i.e. a platform feature
- we want to encourage the adoption and creatiion of library-specific css-in-js custom syntaxes